src/applications/personalization/profile/tests/e2e/address-validation/page-objects/AddressPage.js

```diff
@@ -183,72 +158,53 @@ const validateFocusedElement = element => {
 class AddressPage {
   loadPage = config => {
     setUp(config);
+    Cypress.config({
+      force: true,
+      waitForAnimations: true,
+      scrollBehavior: 'nearest',
+    });
   };
 
   fillAddressForm = fields => {
-    fields.country && cy.findByLabelText(/Country/i).select(fields.country);
-    fields.military &&
-      cy
-        .findByRole('checkbox', {
-          name: /I live on a.*military base/i,
-        })
-        .check();
-    fields.military && cy.get('#root_city').select('FPO');
+    if (fields.military) {
+      cy.selectVaCheckbox('root_view:livesOnMilitaryBase', true);
+      cy.selectVaSelect('root_city', 'FPO');
+    }
+
+    if (fields.country) {
+      cy.selectVaSelect('root_countryCodeIso3', fields.country);
+    }
 
     if (fields.address) {
-      cy.findByLabelText(/^street address \(/i).as('address1');
-      cy.get('@address1').click();
-      cy.get('@address1').clear();
-      cy.findByLabelText(/^street address \(/i).type(fields.address);
+      cy.fillVaTextInput('root_addressLine1', fields.address);
     }
 
     if (fields.address2) {
-      cy.findByLabelText(/^street address line 2/i).as('address2');
-      cy.get('@address2').click();
-      cy.get('@address2').clear();
-      cy.get('@address2').type(fields.address2);
+      cy.fillVaTextInput('root_addressLine2', fields.address2);
     }
 
     if (fields.address3) {
-      cy.findByLabelText(/^street address line 3/i).as('address3');
-
-      cy.get('@address3').click();
-      cy.get('@address3').clear();
-      cy.get('@address3').type(fields.address3);
+      cy.fillVaTextInput('root_addressLine3', fields.address3);
     }
 
-    if (fields.city) {
-      cy.findByLabelText(/City/i).as('city');
-
-      cy.get('@city').click();
-      cy.get('@city').clear();
-      cy.get('@city').type(fields.city);
+    if (fields.city && !fields.military) {
```

Comment:
why did we add the logic `!fields.military` here?

Reply:
The city field changes to a select when you check the lives on a military base checkbox. Line 169 is already setting the city select.